spp_change_request/models/change_request.py

```diff
@@ -152,6 +152,24 @@ def _default_name(self):
         compute="_compute_validation_group_id",
         store=True,
     )
+    hide_button_css = fields.Html(
+        string="CSS", sanitize=False, compute="_compute_hide_button_css", store=False
+    )
+
+    @api.depends("state")
+    def _compute_hide_button_css(self):
+        for record in self:
+            if (
+                record.state == "draft"
+                and record.create_uid != self.env.user
+                and record.assign_to_id != self.env.user
+            ):
+                # Hide edit button when state is 'draft' and current user is not in either create_uid or assign_to_id
```

Comment:
@reichie020212 this logic tell me that: if the record state is "draft" && the creator and assignee must be the current user to edit this record?